utils/gear-replay-cli/README.md

```diff
@@ -108,12 +108,33 @@ Another difference from the `try-runtime` API is that in `gear-replay-cli` the b
 
 In order to use the native runtime build make sure the node is built with the Runtime spec version that matches the one currently uploaded on chain.
 
-* Replay a block on Vara live chain
+General command format and available subcommand are:
+
+```bash
+    gear-replay-cli -h 
+    Commands of `gear-reply` CLI
```

Comment:
Typo?
```suggestion
    Commands of `gear-replay` CLI
```

Reply:
I think the name of the tool should be changed again, now that it not only replays blocks but applies individual extrinsics and soon will be useful also for more precise gas calculation. Good catch anyway.